config.toml

```diff
@@ -26,115 +26,122 @@ disableKinds = ["taxonomy", "taxonomyTerm"]
+  [[params.quicklinks]]
+  title = "Chart Guide"
+  description = "Practical steps for building Helm charts—how to structure, sign, and sync your charts."
+  url = "/docs/topics/charts"
 
-[[params.quicklinks]]
-title = "Video: Intro to Helm"
-description = "Watch Matt Farina and Josh Dolitsky present an introduction to Helm at KubeCon 2019."
-url = "https://www.youtube.com/watch?v=Zzwq9FmZdsU"
+  [[params.quicklinks]]
+  title = "Video: Intro to Helm"
+  description = "Watch Matt Farina and Josh Dolitsky present an introduction to Helm at KubeCon 2019."
+  url = "https://www.youtube.com/watch?v=Zzwq9FmZdsU"
 
-[[params.quicklinks]]
-title = "Migrating from v2 to v3"
-description = "Read our blog post on how to migrate from Helm v2 to Helm v3."
-url = "/blog/migrate-from-helm-v2-to-helm-v3/"
+  [[params.quicklinks]]
+  title = "Migrating from v2 to v3"
+  description = "Read our blog post on how to migrate from Helm v2 to Helm v3."
+  url = "/blog/migrate-from-helm-v2-to-helm-v3/"
 
-[[params.quicklinks]]
-title = "Helm Security Audit"
-description = "Helm has been audited and deemed as recommended for public deployment  during a third-party security audit funded by the CNCF."
-url = "/blog/2019-11-04-helm-security-audit-results/"
+  [[params.quicklinks]]
+  title = "Helm Security Audit"
+  description = "Helm has been audited and deemed as recommended for public deployment  during a third-party security audit funded by the CNCF."
+  url = "/blog/2019-11-04-helm-security-audit-results/"
 
 # Multilingual configuration
 defaultContentLanguage = "en"
 disableLanguages = ["zh"]
 
-# English
-[languages.en]
-title = "Helm"
-description = "Helm - The Kubernetes Package Manager."
-contentDir = "content/en"
-languageName = "English"
-weight = 1
-
-[[languages.en.menus.main]]
-name = "Get Helm"
-url = "https://github.com/helm/helm"
-weight = 1
-
-[[languages.en.menus.main]]
-name = "Blog"
-url = "/blog"
-weight = 2
-
-[[languages.en.menus.main]]
-name = "Docs"
-url = "/docs"
-weight = 3
-
-[[languages.en.menus.main]]
-name = "Charts"
-url = "https://hub.helm.sh/"
-weight = 4
-
-# Chinese
-[languages.zh]
-title = "Helm"
-description = "Helm - The Kubernetes Package Manager."
-contentDir = "content/zh"
-languageName = "中文 Chinese"
-weight = 1
-
-[[languages.zh.menus.main]]
-name = "Get Helm"
-url = "https://github.com/helm/helm"
-weight = 1
-
-#[[languages.zh.menus.main]]
-#name = "Blog"
-#url = "/blog"
-#weight = 2
-
-[[languages.zh.menus.main]]
-name = "Docs"
-url = "/docs"
-weight = 3
-
-[[languages.zh.menus.main]]
-name = "Charts"
-url = "https://hub.helm.sh/"
-weight = 4
-
-[languages.zh.params]
-language_alternatives = ["en"]
-
-[markup]
-  [markup.highlight]
-    style = "manni"
-    guessSyntax = true
\ No newline at end of file
+  # English
+  [languages.en]
+  title = "Helm"
+  description = "Helm - The Kubernetes Package Manager."
+  contentDir = "content/en"
+  languageName = "English"
+  weight = 1
+
+    [[languages.en.menus.main]]
```

Comment:
Nit: TOML is designed around the principle that indentation is not necessary. No need to change in this PR but I'd advise removing this unnecessary complexity.